substrate/scripts/fork-test/README.md

```diff
@@ -12,12 +12,12 @@
 
 1. Check current testnet version.
 2. Checkout repository for this version. For example, if testnet has runtime in version `v27.0.0`, checkout tag `v27.0.0`
-3. Build binary with embedded testnet runtime: `cargo build --release -F testnet-runtime`:
+3. Build binary with embedded testnet runtime: `cargo build --release --features testnet-runtime --features metadata-hash`:
   * copy it to tmp to make it easier: `cp ./target/release/substrate-node ./substrate/scripts/fork-test/liberland-fork-substrate/tmp/liberland-testnet-v27.0.0`
 4. Download runtime you want to test from [GitHub Releases](https://github.com/liberland/liberland_substrate/releases)
   * copy it to tmp to make it easier:  `cp bastiat-v28.0.0.wasm ./liberland-fork-substrate/tmp/bastiat-v28.0.0.wasm`
   * if building runtime yourself,
-    * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release -F testnet-runtime`
+    * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release --features testnet-runtime --features metadata-hash`
```

Comment:
How about going straight for srtool?

```suggestion
    * Checkout the branch with the newer version to test (v28 in this example)and build it `env SRTOOL_TAG=1.81.0 RUNTIME_DIR=./substrate/bin/node/runtime PACKAGE=kitchensink-runtime BUILD_OPTS='"-F testnet-runtime,metadata-hash"' srtool build`
```

Reply:
The path to wasm will be different - it will be in `substrate/bin/node/runtime/target/srtool/release/wbuild/kitchensink-runtime/kitchensink_runtime.compact.compressed.wasm`